content/courses/ada_2022_whats_new/chapters/assignment_target.rst

```diff
@@ -0,0 +1,140 @@
+.. include:: ../../global.txt
+
+Target Name Symbol (@)
+======================
+
+.. note::
+
+    Target name symbol is supported by
+
+    * GNAT Community Edition 2019
+    * GCC 9
+
+Ada 2022 introduces a new symbol :ada:`@`. It could appears only at the
+right side of an assignment statement. This symbol works as a name of
+the left side of the assignment statement. It was introduced to avoid
+code duplication. Instead of retyping (potentially long) name you can
+just use :ada:`@` wherever you need it.
+
+The target name symbol denotes a constant, so you can't pass it into
+[:ada:`in`] :ada:`out` argument of a function.
+
+As an example, let's calculate some statistic for My_Data array:
+
+.. code-block:: ada
```

Comment:
These code blocks should be converted to actual code snippets (using the widgets). By doing this, we not only get the benefit of having the runtime output, but we also avoid the current issues with syntax highlighting.